simbad/mr/molrep_mr.py

```diff
@@ -87,6 +88,7 @@ def __init__(self, hklin, logfile, nmol, pdbin, pdbout, sgalternative, space_gro
         self._space_group = None
 
         self.hklin = hklin
+        self.hklout = hklout
```

Comment:
Definition of a class attribute like this and the `property` does not what you expect it to do.

In fact, the `self.hklout` will never change because you assign anything from your setter/getter call to `self._hklout`. I would encourage to not use all those setters/getters for attributes where you don't do anything. Just use the `self.hklout` type. If there is ever anything that needs extra control, then the setter/getter should be implemented ...